### DISCUSSION TITLE
NextAuth with PWA - email is not opening PWA but browser (Safari)

### DISCUSSION THREAD

**Discussion by dohomi** [nextauthjs/next-auth#3703] [Help]
### Question 💬

I use `next-pwa` and enabled the EmailProvider option of next-auth.

If I try to login via the installed PWA, I don't see how to open the PWA with the login credentials. Only Safari opens (everything works) but the PWA does not. 

### How to reproduce ☕️

After install next-pwa install the app on a device and try to login via `EmailProvider` option.

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by Dev-rick** [comment]
Hi, we are having the same issue on IOS. Did you find a workaround?

Thank you!
